content/en/docs/concepts/services-networking/service.md

```diff
@@ -18,22 +18,23 @@ weight: 10
 
 {{< glossary_definition term_id="service" length="short" >}}
 
-With Kubernetes you don't need to modify your application to use an unfamiliar service discovery mechanism.
-Kubernetes gives Pods their own IP addresses and a single DNS name for a set of Pods,
-and can load-balance across them.
+A key aim of Services in Kubernetes is that you don't need to modify your existing
+application to use an unfamiliar service discovery mechanism.
+You can run code in Pods, whether this is a code designed for a cloud-native world, or
+an older app you've containerized. You use a Service to make that set of Pods available
+on the network so that clients can interact with it.
 
-<!-- body -->
-
-## Motivation
+If you use a {{< glossary_tooltip term_id="deployment" >}} to run your app,
+that Deployment can create and destroy Pods dynamically, as needed - whether that
+is to handle failure, or to match against demand (horizontal scaling).
```

Comment:
I'd suggest we avoid digging into that hole ... horizontal scaling is very complex a topic,
a Deployment cannot handle it at all.
We can mention 'rolling update' which is very relevant to Deployment, if you want.

Reply:
Doesn't the Deployment create the replica set, and the replica set ensure the specified number of pods are created?

Reply:
> Doesn't the Deployment create the replica set, and the replica set ensure the specified number of pods are created?

Yes. It does. But a Deployment doesn't scale the ReplicaSet to handle changes in requests. That is why we need a HPA.

Reply:
So I was asking about the original content that's there currently and agree with you. We need to avoid the rabbit hole and phrase it correctly :)

Reply:
The new wording doesn't mention scaling a workload horizontally.